ui/src/components/Tasks/Characterisation.js

```diff
@@ -448,16 +451,16 @@ Characterisation = connect((state) => {
       beam_size: state.sampleview.currentAperture,
       resolution: (state.taskForm.sampleIds.constructor !== Array
         ? state.taskForm.taskData.parameters.resolution
-        : state.beamline.hardwareObjects.resolution.value),
+        : toFixed(state, "resolution")),
```

Comment:
I dont understand this change, from where the values comes now?

Reply:
i meant the actual values written to the fields, I double checked and now I see the toFixed method, so all good

Reply:
:) cool